components/header.js

```diff
@@ -11,6 +43,106 @@ const Header = () => {
           <li><button>Dark Mode</button></li>
         </ul>
       </nav>
+      <div className={pageStyles.rules}>
+        <div className={pageStyles.cardRanks}>
+          <span>
```

Comment:
it's better practice to not put nested html elements inside a `<span>` and instead to use a `<div>` if you're going to have nested elements